swiftsimio/writer.py

```diff
@@ -200,6 +201,69 @@ def write_particle_group(self, file_handle: h5py.File, compress: bool):
 
         return
 
+    def write_particle_group_metadata(self, file_handle: h5py.File, dset_attributes):
```

Comment:
What are dset attributes? What type does it have?

Can we have a docstring here please.

Reply:
done